craft.lic

```diff
@@ -342,11 +342,11 @@ class Craft
     elsif rank <= 300 # Tier 5 - Basic
       # Buy an abolution and induction sigils and wood totem
       ensure_copper_on_hand(2000, @settings)
-      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 1)
-      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
+      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 1) unless DRCI.exists?("induction sigil-scroll")
+      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 2) unless DRCI.exists?("abolition sigil-scroll")
```

Comment:
Should this be 'abolution' per line 343, or is the typo up there?

Reply:
Let me check that.

Reply:
Abolition is correct.